website/project/views/node.py

```diff
@@ -856,7 +856,7 @@ def private_link_table(node, **kwargs):
 @must_be_valid_project
 def get_editable_children(auth, node, **kwargs):
 
-    if not node.can_edit(auth):
+    if not node.has_permission(auth.user, 'admin'):
```

Comment:
^ if we do that, then I think we can just add @must_have_permission(WRITE) (rather than ADMIN) to the view. One side-effect is that users will only be able to create VOLs for child nodes that the current user has admin access to. I think this is the desired behavior anyway. @jlcohoon correct me if I'm wrong.